examples/ads.amp.html

```diff
@@ -1084,6 +1084,15 @@ <h2 class="broken">GMOSSP 320x50 banner</h2>
       data-mediaid="popin_amp">
   </amp-ad>
 
+  <h2>Postquare widget</h2>
+  <amp-embed width="200" height="200"
```

Comment:
FYI: height 200 here get your example cut in half. It's not a problem for the example here, but want to let you know incase you want to have it fixed.